### PR TITLE
1.3 Update

### PR DESCRIPTION
**Pull request by APickledWalrus**
This is the main part of the next update.

It includes support for registering placeholders with MVdWPlaceholderAPI.
This update also removes the requirement of PlaceholderAPI, if you only use MVdWPlaceholderAPI.

I still need to finish some stuff up including the wiki, but other than that the update is mostly ready.

### CODE REVIEW

File: src/main/java/io/github/apickledwalrus/placeholderaddon/Main.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	public static boolean hasPapi() {
+		return hasPapi;
+	}
+
+	@Override
+	public void onEnable() {
+		instance = this;
+		if (!hasMVdW && !hasPapi) {
+			Logger.getLogger("Minecraft").severe("[skript-placeholders] No placeholders plugin found! Disabling!");
+			Logger.getLogger("Minecraft").severe("[skript-placeholders] Has PlaceholderAPI: " + hasPapi + " Has MVdWPlaceholderAPI: " + hasMVdW);
```
Comment (JRoy) [inline src/main/java/io/github/apickledwalrus/placeholderaddon/Main.java]: You already know that `hasMVdW` and `hasPapi` are false, why do you need this message?

File: src/main/java/io/github/apickledwalrus/placeholderaddon/mvdwapi/MvdwAPIListener.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+					@Override
+					public String onPlaceholderReplace(PlaceholderReplaceEvent event) {
+						MvdwAPIEvent e = new MvdwAPIEvent(event.getPlayer(), event.getPlaceholder());
+							Bukkit.getServer().getPluginManager().callEvent(e);
+							return e.getResult();
+				}
+			
+				});
+	}
+
```
Comment (JRoy) [inline src/main/java/io/github/apickledwalrus/placeholderaddon/mvdwapi/MvdwAPIListener.java]: Excess whitespace

File: src/main/java/io/github/apickledwalrus/placeholderaddon/mvdwapi/MvdwAPIListener.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+import be.maximvdw.placeholderapi.PlaceholderReplaceEvent;
+import be.maximvdw.placeholderapi.PlaceholderReplacer;
+import io.github.apickledwalrus.placeholderaddon.mvdwapi.MvdwAPIEvent;
+
+public class MvdwAPIListener {
+
+	public static void registerPlaceholder(Plugin plugin, String placeholder) {
+		PlaceholderAPI.registerPlaceholder(plugin, placeholder,
+				new PlaceholderReplacer() {
+
```
Comment (JRoy) [inline src/main/java/io/github/apickledwalrus/placeholderaddon/mvdwapi/MvdwAPIListener.java]: Excess whitespace

File: src/main/java/io/github/apickledwalrus/placeholderaddon/Main.java (hunk 1 above)
Comment (APickledWalrus) [reply src/main/java/io/github/apickledwalrus/placeholderaddon/Main.java]: good question

**Pull request merged**